### PR TITLE
AC Automated Test - LRAC-11361 | master

### PR DESCRIPTION
**Pull request by calvinctvo**
https://issues.liferay.com/browse/LRAC-11361 (Automation ticket)
https://issues.liferay.com/browse/LRAC-11360 (Bug Ticket)
*Note: LRAC-11360 still waiting to be fix

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase [hunk 1]
```diff
@@ -18,6 +18,36 @@ definition {
...
+			ACUtils.changePagination(itemsPerPage = "5");
+		}
+
+		task ("Click next page") {
+			ACUtils.checkPreviousNextButton(nextButton = "true");
+		}
+
+		task ("Check that the existing properties in the AC appear in the list") {
+			ACUtils.assertItemsName(itemList = "Property Name 1");
+		}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase:48]: Hey @calvinctvo, very nice work here. I think you could check that the first page has properties 1,2,3,4,5 and that property 6 does not appear, so then you switch to page 2 and verify that only property 6 appears and that the others not. What do you think?

Comment (calvinctvo) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase:48]: Hey @ViniLopes20 , I think that can work. But also, I ran the test and property name added in created order, so Property 1 would be in the bottom.

**Review by ViniLopes20 — APPROVED** [review]
LGTM, thanks Calvin 🎉

**Review by manuferreira — APPROVED** [review]
LGTM, nice work Calvin :tada:

**Review by joshitagaki — APPROVED** [review]
Nice LGTM! Thanks for working on this

**Comment by calvinctvo** [comment]
ci:forward

**Pull request closed without merging**